core/src/read.jl

```diff
@@ -883,81 +887,123 @@ function Basin(db::DB, config::Config, graph::MetaGraph)::Basin
     return basin
 end
 
+function get_greater_than!(greater_than, conditions_compound_variable, starttime)::Nothing
+    (; node_id) = first(conditions_compound_variable)
+    errors = false
+
+    for condition_group in
+        IterTools.groupby(row -> row.condition_id, conditions_compound_variable)
+        condition_group = StructVector(condition_group)
+
+        if any(ismissing, condition_group.time)
```

Comment:
Can this happen, as we replace missings in the load_structvector? I think the check would be for multiple identical timesteps?